packages/gasket-plugin-happyfeet/lib/index.js

```diff
@@ -0,0 +1,21 @@
+const happyFeet = require('happy-feet');
+
+module.exports = {
+    name: require('../package').name,
+    hooks: {
+        preboot: async function preboot(gasket) {
+            const happyConfig = gasket.config.get('happyFeet') || {}
+            gasket.happyFeet = happyFeet(happyConfig)
+        },
+        healthcheck: async function healthcheck(gasket, HealthCheckError) {
+            const happyConfig = gasket.config.get('happyFeet') || {}
+            const happy = gasket.happyFeet;
+            if (happy.state === happy.STATE.UNHAPPY) {
+                // flag pod to be removed from LB
+                throw new HealthCheckError(`Pod has exceeded its memory limits \n 
```

Comment:
There can be other reasons HappyFeet flags as unhappy besides this. I'd keep the message generic. `happy-feet` does log the reason when its state changes, so the logs should cover debugging issues.